.github/workflows/spellchecker.yml

```diff
@@ -0,0 +1,24 @@
+name: spellchecker
+on:
+  push:
+    branches:
+      - master
```

Comment:
PR is enough 😊

Reply:
Hmmm. I reference the PR https://github.com/apache/apisix/pull/3011.
I think it is better to keep both `push` and `pr`.

Reply:
🤔 but when there is a mistake spelling after merging master?

Reply:
The aim is to avoid the mistake of spelling. 
So we shouldn't make a mistake of spelling even if pushing to master.

Reply:
There has 1 protection rule to forbidden push directly.

Reply:
Got it. I will delete it.